Author: Davide Grassano (@Crivella) - CECAM HQ
Addresses issue from https://github.com/HDFGroup/hdf5/issues/5660 where the fortran wrappers are not passing the include
directory due to it being stripped in `pkg-config`.

diff --git a/config/libh5cc.in b/config/libh5cc.in
index b16de1d8729..a42dc5a2d52 100644
--- a/config/libh5cc.in
+++ b/config/libh5cc.in
@@ -230,7 +230,7 @@ if test "${USE_HL_LIB}" = "yes"; then
 fi
 
 # Evaluate pkg-config command for later use
-pc_flags="`pkg-config $pc_args`"
+pc_flags="`PKG_CONFIG_ALLOW_SYSTEM_CFLAGS=1 pkg-config $pc_args`"
 if test $? -ne 0; then
   echo "pkg-config failed with arguments: $pc_args" >&2
   exit $EXIT_FAILURE
